pkg/users/local.js

```diff
@@ -271,6 +271,35 @@ function is_user_in_group(user, group) {
     return false;
 }
 
+function change_shell(user, shell) {
+    var dfd = cockpit.defer();
+
+    cockpit.spawn([ "chsh", "-s", shell, user ], {superuser: "require", err: "out" })
```

Comment:
For what you want to do, `err: "message"` is probably more appropriate? (see https://cockpit-project.org/guide/latest/cockpit-spawn.html)